src/main/java/tictactoe/Context.java

```diff
@@ -4,47 +4,46 @@
 
 public class Context {
 
-	private Aktion splash;
-	private Aktion xTurn;
-	private Aktion oTurn;
-	private Aktion ende;
-	private Aktion currentState;
+	private Action splash;
+	private Action xTurn;
+	private Action oTurn;
+	private Action end;
+	private Action currentState;
 	private Board board;
```

Comment:
If not needed otherwise I would set the fields to final, since they are not changed - excepted currentState - if I have paid attention correctly.

```suggestion
	private final Action splash;
	private final Action xTurn;
	private final Action oTurn;
	private final Action end;
	private Action currentState;
	private final Board board;
```